src/main/java/io/github/lucasstarsz/fastj/math/Maths.java

```diff
@@ -136,4 +136,17 @@ public static float magnitude(Point p) {
     public static boolean floatEquals(float a, float b) {
         return Math.abs(a - b) < FloatPrecision;
     }
+
+    /**
+     * Linearly interpolates the value of lerping between x and y, by the interpolation point t.
```

Comment:
Note that your documentation has become a bit redundant there -- try and clean it up.